x-pack/plugins/integration_assistant/common/api/model/common_attributes.ts

```diff
@@ -160,10 +160,10 @@ export const Integration = z.object({
  * An array of pipeline results.
  */
 export type PipelineResults = z.infer<typeof PipelineResults>;
-export const PipelineResults = z.array(z.object({}));
+export const PipelineResults = z.array(z.object({}).passthrough());
 
 /**
  * An array of errors.
  */
 export type Errors = z.infer<typeof Errors>;
-export const Errors = z.array(z.object({}));
+export const Errors = z.array(z.any());
```

Comment:
`z.array(z.any())`:
![image](https://github.com/elastic/kibana/assets/1625373/ab59183d-bed4-40ad-8ab0-451adea1446c)

`z.array(z.unknown())`:
![image](https://github.com/elastic/kibana/assets/1625373/793944b2-9afc-4f0a-8c1e-fb1798701a62)

Wdyt?

Reply:
@kapral18 thanks for the suggestion, I realized `errors` entry is not necessary. I didn't want to pollute this PR so I rolled back that change and created a separate PR [here](https://github.com/elastic/kibana/pull/187050).